### PR TITLE
Changelog for 2.15.0

### PR DESCRIPTION
**Pull request by Didainius**
This PR adds changelog for `2.15.0` release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,110 @@
...
+* Deprecated `mediaRecord.DeleteMetadataAsync` in favor of `mediaRecord.DeleteMetadataEntryAsync` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Deprecated `media.AddMetadata` in favor of `media.AddMetadataEntry` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Deprecated `media.AddMetadataAsync` in favor of `media.AddMetadataEntryAsync` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Deprecated `media.DeleteMetadata` in favor of `media.DeleteMetadataEntry` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Deprecated `media.DeleteMetadataAsync` in favor of `media.DeleteMetadataEntryAsync` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Deprecated `vm.ChangeMemorySize` in favor of `vm.ChangeMemory` ([#432](https://github.com/vmware/go-vcloud-director/pull/432))
+* Deprecated `vm.ChangeCPUCount` and `vm.ChangeCPUCountWithCore` in favor of `vm.ChangeCPU` ([#432](https://github.com/vmware/go-vcloud-director/pull/432))
+
+## NOTES
+* Bump `staticcheck` version to 2022.1 with Go 1.18 support ([#457](https://github.com/vmware/go-vcloud-director/pull/457))
```
Comment (vbauzysvmware) [inline CHANGELOG.md:107]: empty line missing?

**Review by vbauzysvmware — APPROVED** [review]
This is huge list :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,110 @@
 ## 2.15.0 (TBC)
 
-Changes in progress for v2.15.0 are available at [.changes/v2.15.0](https://github.com/vmware/go-vcloud-director/tree/main/.changes/v2.15.0) until the release.
+## FEATURES
+* Added support for Shareable disks, i.e., independent disks that can be attached to multiple VMs which is available from
+  API v35.0 onwards. Also added uuid to the Disk structure which is a new member that is returned from v36.0 onwards. This
```
Comment (mikeletux) [inline CHANGELOG.md:5]: UUID is written in lowercase here an later in capital letters. Let's be consistent with this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,110 @@
...
+  member holds a UUID that can be used to correlate the disk that is attached to a particular VM from the VCD side and the
+  VM host side. ([#383](https://github.com/vmware/go-vcloud-director/pull/383))
+* Added support for uploading OVF using URL `catalog.UploadOvfByLink` ([#422](https://github.com/vmware/go-vcloud-director/pull/422)], [[#426](https://github.com/vmware/go-vcloud-director/pull/426))
+* Added support for updating vApp template `vAppTemplate.UpdateAsync` and `vAppTemplate.Update` ([#422](https://github.com/vmware/go-vcloud-director/pull/422))
+* Add methods `catalog.PublishToExternalOrganizations` and `adminCatalog.PublishToExternalOrganizations` ([#424](https://github.com/vmware/go-vcloud-director/pull/424))
```
Comment (mikeletux) [inline CHANGELOG.md:10]: `Add` and `Added`are used interchangeably.  Maybe we should use one or another, not both to keep consistency?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,110 @@
...
+* Added types `types.MetadataStringValue`, `types.MetadataNumberValue`, `types.MetadataDateTimeValue` and `types.MetadataBooleanValue` 
+  for adding different kind of metadata to entities ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Added support to set, get and delete metadata to AdminCatalog with the methods 
+  `AdminCatalog.AddMetadataEntry`, `AdminCatalog.AddMetadataEntryAsync`, `AdminCatalog.GetMetadata`, 
+  `AdminCatalog.DeleteMetadataEntry` and `AdminCatalog.DeleteMetadataEntryAsync`. ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Added support to get metadata from Catalog with method `Catalog.GetMetadata` ([#430](https://github.com/vmware/go-vcloud-director/pull/430))
+* Added to *VM* and *VApp* the methods `DeleteMetadataEntry`, `DeleteMetadataEntryAsync`, `AddMetadataEntry` and `AddMetadataEntryAsync`
```
Comment (mikeletux) [inline CHANGELOG.md:17]: Maybe we should get rid of asterisks here?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,110 @@
...
+  `catalog`, `catalog item`, `edge gateway`, `independent disk`, `media`, `network`, `org`, `PVDC`, `PVDC storage profile`, `vApp`, `vApp template`,`VDC` and `VDC storage profile`;
+  with the methods
+  `VCDClient.GetMetadataByHref`, `VCDClient.AddMetadataEntryByHref`, `VCDClient.AddMetadataEntryByHrefAsync`,
+  `VCDClient.DeleteMetadataEntryByHref` and `VCDClient.DeleteMetadataEntryByHrefAsync` ([#454](https://github.com/vmware/go-vcloud-director/pull/454))
+* Added functions `VdcGroup.GetOpenApiOrgVdcNetworkById` and `VdcGroup.CreateOpenApiOrgVdcNetwork` ([#456](https://github.com/vmware/go-vcloud-director/pull/456))
+
+## IMPROVEMENTS
+* Bump Default API Version to V35.0 ([#434](https://github.com/vmware/go-vcloud-director/pull/434))
+* Disk methods have now the ability to access new properties from API version 36.0. They are: `DiskRecordType.SharingType`, `DiskRecordType.UUID`, `DiskRecordType.Encrypted`, `Disk.SharingType`, `Disk.UUID` and `Disk.Encrypted` ([#436](https://github.com/vmware/go-vcloud-director/pull/436))
+* New method added `Disk.GetAttachedVmsHrefs` ([#436](https://github.com/vmware/go-vcloud-director/pull/436))
```
Comment (mikeletux) [inline CHANGELOG.md:55]: Shouldn't `Disk.GetAttachedVmsHrefs` get moved into `Features`?

**Review by mikeletux — APPROVED** [review]
Just some few comments

File: CHANGELOG.md (hunk 2 above)
Comment (Didainius) [reply CHANGELOG.md:5]: Made uppercase for both

File: CHANGELOG.md (hunk 3 above)
Comment (Didainius) [reply CHANGELOG.md:10]: Made past tense everywhere

File: CHANGELOG.md (hunk 4 above)
Comment (Didainius) [reply CHANGELOG.md:17]: replaced with `

File: CHANGELOG.md (hunk 1 above)
Comment (Didainius) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 5 above)
Comment (Didainius) [reply CHANGELOG.md]: The line is a lot thinner here than for Terraform - moved it.

**Review by lvirbalas — APPROVED** [review]
Looks great!

**Pull request merged**